examples/golang/docker-compose.yaml

```diff
@@ -0,0 +1,18 @@
+version: "3.8"
+services:
+
+  proxy:
+    build:
+      context: .
+      dockerfile: Dockerfile-proxy
+    depends_on:
+      web_service:
+        condition: service_healthy
+    ports:
+    - "${PORT_PROXY:-10000}:10000"
+
+  web_service:
+    environment:
+    - PORT=9000
+    build:
+      context: ../shared/echo
```

Comment:
apologies, i assumed this was using the `shared/flask` service which has a healthcheck for this reason

```console
ERROR: for proxy  Service "web_service" is missing a healthcheck configuration
Encountered errors while bringing up the project.
ERROR: starting golang .
```

i reckon lets use the flask container if we can

Reply:
this seems to need to add a service.py. I'll try it.

Reply:
yep - it does - be sure to use port `8080` as that is healthchecked

im just working on a pr now to clean up some of the other sandbox healthchecks/dependencies

Reply:
cool